tags/outputs.tf

```diff
@@ -1,9 +1,9 @@
 output "ids" {
-  description = "The id of the tag."
+  description = "The list of id of the new tags."
   value       = digitalocean_tag.this.*.id
 }
 
 output "names" {
-  description = "The name of the tag."
-  value       = digitalocean_tag.this.*.name
+  description = "The list of name of the tags."
```

Comment:
please use: The names of the tags. Includes all the tags specified, including the pre-existing ones.